lib/matplotlib/backend_bases.py

```diff
@@ -912,13 +910,6 @@ def get_joinstyle(self):
         """
         return self._joinstyle
 
-    def get_linestyle(self, style):
```

Comment:
In https://github.com/matplotlib/matplotlib/pull/7730#issuecomment-270044316, you were fine changing the signature of `get_linestyle` (which would be a breaking change for anyone (hem) relying on this method) without a deprecation cycle, is it clear that removing the method is any different?